include/utils/ExtrusionLine.h

```diff
@@ -228,47 +228,6 @@ struct ExtrusionLine
 
         return ret;
     }
-
-    /*!
-     * Create a true-extrusion area shape for the path; this means that each junction follows the bead-width
-     * set for that junction.
-     */
-    Polygons toExtrusionPolygons() const
```

Comment:
Why was this deleted, I think this can still come in handy, no?

Reply:
I you decide to re-add it to the code base add the`[[maybe_unused]]` attribute so it won't result in a compiler warning